### PR TITLE
Remove Tray in focus and full screen modes

### LINKED ISSUE #103: Tray should be hidden when in focus mode and fullscreen
STR:
1) Go fullscreen or focus mode

Expect results: tray is hidden
Actual results: tray is still visible

### PR DESCRIPTION
**Pull request by keianhzo**
Fixes #103

### CODE REVIEW

File: app/src/common/shared/org/mozilla/vrbrowser/WidgetManagerDelegate.java [hunk 1]
```diff
@@ -18,4 +18,5 @@
     void popBackHandler(Runnable aRunnable);
     void fadeOutWorld();
     void fadeInWorld();
+    void setTrayVisible(boolean visible);
```
Comment (MortimerGoro) [inline app/src/common/shared/org/mozilla/vrbrowser/WidgetManagerDelegate.java]: Tray already had a method to make it visible or invisible: Toggle(bool aEnabled)

File: app/src/main/cpp/BrowserWorld.cpp [hunk 2]
```diff
@@ -1114,6 +1114,18 @@ BrowserWorld::CreateTray() {
   m.tray->SetTransform(transform);
 }
 
+void
+BrowserWorld::SetTrayVisible(bool visible) const {
+  if (!m.tray)
```
Comment (MortimerGoro) [inline app/src/main/cpp/BrowserWorld.cpp]: Tray already had a method to make it visible or invisible: tray->Toggle(bool aEnabled)

**Comment by keianhzo** [comment]
Fixed

File: app/src/main/cpp/BrowserWorld.cpp (hunk 2 above)
Comment (keianhzo) [reply app/src/main/cpp/BrowserWorld.cpp]: Fixed

**Comment by MortimerGoro** [comment]
@keianhzo we need to fix some conflicts. Sorry for the branch deletion, it was a mistake ;)

**Pull request merged**